packages/block-library/src/navigation/style.scss

```diff
@@ -1,15 +1,25 @@
+// Default background and font color
+.wp-block-navigation:not(.has-background) .wp-block-navigation__container {
+	.wp-block-navigation__container {
+		// Set a background color for submenus so that they're not transparent.
+		// NOTE TO DEVS - if refactoring this code, please double-check that
+		// submenus have a default background color, this feature has regressed
+		// several times, so care needs to be taken.
+		background-color: $white;
+		color: $gray-900;
+	}
+}
+
+.wp-block-navigation .wp-block-navigation__container {
+	.wp-block-navigation__container {
+		align-items: stretch;
```

Comment:
I don't think this'll have any effect as the element isn't set to flex or grid display.

Reply:
This was needed to override this rule that was added in #28836 on navigation containers:
https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/navigation/style.scss#L3

The parent navigation link element is a flex container. 

I've switched it to use `normal`. I did also consider `unset`. Not really sure what is best.